packages/design-system/src/components/CssDebugger/CssDebugger.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const CssDebugger = () => {
+  React.useEffect(() => {
+    window.addEventListener('load', handleRefresh);
+
+    return () => {
+      window.removeEventListener('load', handleRefresh);
+    };
+  }, []);
+
+  const handleRefresh = () => {
+    if (process.env.NODE_ENV !== 'production') {
+      document.documentElement.style.setProperty('--debugger', 'true');
+    }
```

Comment:
what do you mean by the root scope of the module?

Reply:
Update: We talked through this and found that my idea of putting it at the root scope of the module would cause the module to have side-effects, and our core package currently doesn't have side-effects. We want to keep it that way.